browser/src/Editor/NeovimEditor.tsx

```diff
@@ -467,40 +481,57 @@ export class NeovimEditor extends Editor implements IEditor {
         }
     }
 
-    private _onVimEvent(eventName: string, evt: EventContext): void {
-        UI.Actions.setWindowCursor(evt.windowNumber, evt.line - 1, evt.column - 1)
+    private _updateWindow(currentBuffer: EventContext) {
```

Comment:
It looks like this is another case where these methods - `UI.Actions.setWindowCursor` and `this._syntaxHighlighter.notifyViewportChanged` - aren't going to be called as frequently as before. Prior to this change, they were called for every event. The `UI.Actions.setWindowCursor` is particularly sensitive because there are lots of UI pieces that depend on it being set (ie, all the quick info and context menus).

I think it's risky to change this behavior as part of this change - could we make `this._updateWindow` called for every event, as before? Similiar to what you did for `this._bufferManager.updateBufferFromEvent`.

Reply:
Ah of course, will do, out curiosity which events trigger this handler or is it a catch all almost for all vim events not handled in the neovim instance switch?